lib/errors/api_error_handling.rb

```diff
@@ -5,7 +5,8 @@ def log_error(error, update_service: nil, sentry_extra: nil)
     Rails.logger.info "Caught #{error}"
     sentry_tags = nil
     if update_service.present?
-      update_service.update_error_stats
+      new_errors_count = sentry_extra&.dig(:new_errors_count) || 1
```

Comment:
We don't actually use this anywhere, correct? So this will not change the current behavior of the related bug until we also new_errors_count within the code that is collating errors?

Reply:
Yes, that is correct. 
This change will not affect the current behavior until we also implement `new_errors_count` when reporting errors in places where errors are batched or where multiple errors may occur. 
This will enable the log_error method to increment the error count based on the actual number of errors, instead of defaulting to 1.